app/controllers/publishers_controller.rb

```diff
@@ -164,7 +164,7 @@ def create_auth_token
     if emailer.perform
       # Success shown in view #create_auth_token
     else
-      flash.now[:alert] = emailer.error
+      flash.now[:alert] = %Q[#{emailer.error[0]}<a href="/">#{emailer.error[1]}</a>].html_safe
```

Comment:
this looks scary to me, i do not think we should commit this.

Instead I suggest following this guide on flash messages ("Identifiers for more complex messages"); add a new flash[:login_link] and new partial, and check and render that in `application.html.slim`.
https://thepugautomatic.com/2012/08/the-rails-flash/